server/backend/api/app/main.py

```diff
@@ -98,31 +100,46 @@ def main():
     return RedirectResponse(url="/docs/")
 
 
-apiPrefix = ''
-app.include_router(router_application.router,
-                   prefix=apiPrefix + '/fam_applications',
-                   tags=["FAM Applications"])
-app.include_router(router_user_role_assignment.router,
-                   prefix=apiPrefix + '/user_role_assignment',
-                   tags=["FAM User Role Assignment"])
-app.include_router(router_forest_client.router,
-                   prefix=apiPrefix + '/forest_clients',
-                   dependencies=[Depends(router_guards.authorize)],
-                   tags=["FAM Forest Clients"])
-app.include_router(router_idim_proxy.router,
-                   prefix=apiPrefix + '/identity_search',
-                   dependencies=[Depends(router_guards.authorize)],
-                   tags=["IDIR/BCeID Proxy"])
+apiPrefix = ""
+app.include_router(
+    router_application.router,
+    prefix=apiPrefix + "/fam_applications",
+    tags=["FAM Applications"],
+)
+app.include_router(
+    router_user_role_assignment.router,
+    prefix=apiPrefix + "/user_role_assignment",
+    tags=["FAM User Role Assignment"],
+)
+app.include_router(
+    router_forest_client.router,
+    prefix=apiPrefix + "/forest_clients",
+    dependencies=[Depends(router_guards.authorize)],
+    tags=["FAM Forest Clients"],
+)
+app.include_router(
+    router_idim_proxy.router,
+    prefix=apiPrefix + "/identity_search",
+    dependencies=[Depends(router_guards.authorize)],
+    tags=["IDIR/BCeID Proxy"],
+)
+app.include_router(
+    router_user_terms_conditions.router,
+    prefix=apiPrefix + "/user_terms_conditions",
+    dependencies=[Depends(router_guards.authorize)],
+    tags=["FAM User Terms and Conditions"],
```

Comment:
Most of the changes in this file is just formatting, added router with basic authorization check